ansible/roles/ocp-workload-pam7-offer-management-dmn-pmml/tasks/rhpam_openshift_dmn_pmml_workload.yml

```diff
@@ -1,60 +1,104 @@
 ---
 
-# TODO: Replace the ImageStream approach with Binary Builds from a Dockerfile that patch the BC and KIE-Server images with PMML support and Prometheus Metrics Provider.
+# TODO: Replace the ImageStream approach with Binary Builds from a Dockerfile that
+# patch the BC and KIE-Server images with PMML support and Prometheus Metrics Provider.
 
 - name: Debug message
   debug:
     msg: "Adding jPMML support to RHPAM 7.5."
 
 - name: Create Business Central DMN PMML build
-  shell: oc new-build --strategy docker --image-stream="openshift/rhpam-businesscentral-rhel8:7.5.0" --binary --name rhpam-businesscentral-rhel8-dmn-pmml -n {{ OCP_PROJECT }}
+  shell: >-
+    oc new-build --strategy docker
+    --image-stream="openshift/rhpam-businesscentral-rhel8:7.5.0" --binary
+    --name rhpam-businesscentral-rhel8-dmn-pmml
+    -n {{ OCP_PROJECT }}
 
 - name: Create KIE-Server DMN PMML build
-  shell: oc new-build --strategy docker --image-stream="openshift/rhpam-kieserver-rhel8:7.5.0" --binary --name rhpam-kieserver-rhel8-dmn-pmml -n {{ OCP_PROJECT }}
+  shell: >-
+    oc new-build --strategy docker
+    --image-stream="openshift/rhpam-kieserver-rhel8:7.5.0"
+    --binary --name rhpam-kieserver-rhel8-dmn-pmml
+    -n {{ OCP_PROJECT }}
 
 - name: "Configure Business Central ImageStream"
-  shell: "oc patch dc/{{ pam_app_name }}-rhpamcentr --type='json' -p '[{\"op\": \"replace\", \"path\": \"/spec/triggers/0/imageChangeParams/from/name\", \"value\": \"rhpam-businesscentral-rhel8-dmn-pmml:latest\"}]' -n {{ OCP_PROJECT }}"
+  shell: >-
+    oc patch dc/{{ pam_app_name }}-rhpamcentr --type='json'
+    -p '[{"op": "replace", "path": "/spec/triggers/0/imageChangeParams/from/name",
```

Comment:
Have you tested removing the escaping of the `"` chars ?

Reply:
Checked via yaml to json conversion. Has not been conventionally tested. Would talk to Jonathan if you feel that is necessary - my time on this project ends on Friday.

Reply:
@jkupferer ^

Reply:
confirmed, looks good